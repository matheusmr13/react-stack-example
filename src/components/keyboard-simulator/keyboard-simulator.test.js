import React from 'react';
import { shallow } from 'enzyme';
import KeyboardSimulator from './index';

describe('KeyboardSimulator component', () => {
	it('should render properly', () => {
		const keyboardSimulator = shallow(<KeyboardSimulator />);
		expect(keyboardSimulator).toMatchSnapshot();
	});
});
