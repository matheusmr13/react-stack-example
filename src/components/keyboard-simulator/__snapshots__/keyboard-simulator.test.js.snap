// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KeyboardSimulator component should render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <KeyboardSimulator />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span
          className="keyboard-simulator__text"
>
           
          <span
                    className="keyboard-simulator__inputing"
          />
</span>,
        <span
          className="keyboard-simulator__cursor keyboard-simulator__cursor--idle"
/>,
      ],
      "className": "keyboard-simulator",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
            " ",
            <span
              className="keyboard-simulator__inputing"
/>,
          ],
          "className": "keyboard-simulator__text",
        },
        "ref": null,
        "rendered": Array [
          undefined,
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "",
              "className": "keyboard-simulator__inputing",
            },
            "ref": null,
            "rendered": "",
            "type": "span",
          },
        ],
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "keyboard-simulator__cursor keyboard-simulator__cursor--idle",
        },
        "ref": null,
        "rendered": null,
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span
            className="keyboard-simulator__text"
>
             
            <span
                        className="keyboard-simulator__inputing"
            />
</span>,
          <span
            className="keyboard-simulator__cursor keyboard-simulator__cursor--idle"
/>,
        ],
        "className": "keyboard-simulator",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              " ",
              <span
                className="keyboard-simulator__inputing"
/>,
            ],
            "className": "keyboard-simulator__text",
          },
          "ref": null,
          "rendered": Array [
            undefined,
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "",
                "className": "keyboard-simulator__inputing",
              },
              "ref": null,
              "rendered": "",
              "type": "span",
            },
          ],
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "keyboard-simulator__cursor keyboard-simulator__cursor--idle",
          },
          "ref": null,
          "rendered": null,
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
